pkg/oc/clusterup/status.go

```diff
@@ -34,18 +36,18 @@ var (
 )
 
 // NewCmdStatus implements the OpenShift cluster status command.
-func NewCmdStatus(name, fullName string, f kcmdutil.Factory, out io.Writer) *cobra.Command {
+func NewCmdStatus(name, fullName string, f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
```

Comment:
nit: same here, NewClientStatusConfig in a future PR

Reply:
Also, are io streams not used in this command either?

Reply:
Same see https://github.com/openshift/origin/pull/20232